packages/vector_graphics_compiler/lib/src/geometry/pattern.dart

```diff
@@ -0,0 +1,29 @@
+import '../../src/svg/resolver.dart';
+import 'matrix.dart';
+
+/// Pattern information for encoding.
+class PatternData {
+  /// Constructs new [PatternData].
+  PatternData(this.x, this.y, this.width, this.height, this.transform);
+
+  /// The x coordinate shift of the pattern tile.
+  double x;
+
+  /// The y coordinate shift of the pattern tile.
+  double y;
+
+  /// The width of the pattern.
```

Comment:
Similar comment as other pattern data class: provide information on validate values and units

Reply:
fixed.